docs/extending/custom_tasks.md

```diff
@@ -4,11 +4,16 @@ The Workflow system allows users to create tasks, which represent stages of mode
 
 Wagtail provides one built-in task type: `GroupApprovalTask`, which allows any user in specific groups to approve or reject moderation.
 
-However, it is possible to implement your own task types. Instances of your custom task can then be created in the `Tasks` section of the Wagtail Admin.
+However, it is possible to implement your own task types. Instances of your custom task can then be created in the `Workflow tasks` section of the Wagtail Admin.
```

Comment:
```suggestion
However, it is possible to implement your own task types. Instances of your custom task can then be created in the workflow tasks section of the Wagtail Admin.
```

If we are not referencing the actual model we should probably avoid the backticks.